### PR TITLE
feat: Implement ModDashboardLeftNavbar with partial functionality.

### LINKED ISSUE #451: Create Sidebar with Dropdown component for the Moderation Panel
# Problem
The Moderation Panel needs a sidebar that follows provided styling. Currently no component exists. A component is necessary and should have 3 tabs as seen on Figma [here](https://www.figma.com/file/QXQxlw4jNFIhMxkKU7bbwq/Find-a-doc%2C-Japan?type=design&node-id=1621-10485&mode=design&t=YlT64bYAuVCI537V-4](https://www.figma.com/file/QXQxlw4jNFIhMxkKU7bbwq/Find-a-doc%2C-Japan?type=design&node-id=1621-17366&mode=design&t=YlT64bYAuVCI537V-4 ) and [here](https://www.figma.com/file/QXQxlw4jNFIhMxkKU7bbwq/Find-a-doc%2C-Japan?type=design&node-id=1621-10485&mode=design&t=GrgnAmRYc3OVrfpY-4 ).

# Success
This component exists with the styling as shown in the Figma linked above. The Sidebar does NOT need full functionality currently

^

Image

### Requirements

- [ ] Component exists with proper text
- [ ] It follows the styling as shown in Figma
- [ ] logs into the console which dropdown is selected

# Implementation notes
From @ermish 

Since this sidebar will be housing dynamic content that will change based on what is displayed in the main page componet, we should consider some design points here. 
- create a parent moderation sidebar component that is agnostic of the content. Something like `moderationSidebarComponent`
- use consistent naming to show that the sidebar content matches the page. 
    - `submissionsModSidebarContent` and `submissionsModPage`
    - and `editSubmissionModSidebarContent` and `editSubmissionModSiderbarPage`
- I think we'll need some for of back button on the sidebar, but we'll need to talk design first. 
- Let's add a side-scolling animation to the sidebar when switching between content. Consider it like going through sub-navbars and swiping back to the parent sidebar content.  (ask me if this is confusing)

### PR DESCRIPTION
**Pull request by NabbeunNabi**
Resolves #451 

## 🔧 What changed
Before there was no LeftNavBar for the dashboard view of the moderation panel. Now it exists with partial functionality. It still needs the functionality with the fetching from the backend. This is mainly a UI component that has functions set up to change the state.

## 📸 Screenshots

-   ### Before

image

-   ### After

image

### CODE REVIEW

File: components/ModDashboardLeftNavbar.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+</template>
+
+<script setup lang="ts">
+import { ref, Ref } from 'vue'
+import noteStackAddSvg from "../assets/images/note-stack-add.svg"
+import checkBoxSvg from "../assets/images/check-box.svg"
+import disabledByDefault from "../assets/images/disabled-by-default.svg"
+
+const modDashboardViewOptions = [{
+    displayText: "Facilities",
```
Comment (theyokohamalife) [inline components/ModDashboardLeftNavbar.vue:30]: Let's wrap all the `displayText` in translatable strings using `$t('')`

**Review by theyokohamalife — CHANGES_REQUESTED** [review]
Looking great so far! Just needs a few small changes.

**Comment by NabbeunNabi** [comment]
Added the translation string as requested. in the console is saying it defaults to the US since there is no Japanese for it at the moment. So I believe it is working as expected

**Review by theyokohamalife — APPROVED** [review]
LGTM ✨

**Pull request merged**